src/phockup.py

```diff
@@ -98,9 +129,14 @@ def get_output_dir(self, date):
 
         fullpath = os.path.sep.join(path)
 
+        lock = threading.Lock()
+        lock.acquire()
```

Comment:
I guess this lock works across all threads? Not sure what will happen if two threads attempt to create the same directory. The first one will succeed, but the second one may fail?

Reply:
I found the `exist_ok=True` parameter on `os.makedirs`, that creates a dir or do nothing if it already exist. Now I guess I have to find how to avoid concurrency to other `os` calls in `process_file` function.

Reply:
I made some changes to the code. What should I do? Create a new commit, amend the changes and force push, or create a new pull request?

Reply:
When you push your changes to your repository it updates the pull request automatically.